recipes/cyarray/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "cyarray" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 00e1271e66e5f82ff846e054a61f78e22e04589253b7705813ea8a624642a402
+  
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - numpy
+  host:
+    - cython >=0.20
+    - numpy
```

Comment:
If this links against numpy, it should be done as here: https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy